src/moaicore/MOAIDraw.h

```diff
@@ -68,6 +69,7 @@ class MOAIDraw :
 	//----------------------------------------------------------------//
 	static void			DrawLuaParams			( lua_State* L, u32 primType );
 	static void			DrawLuaArray			( lua_State* L, u32 primType );
+	static void			DrawFromLua				( lua_State* L, u32 primType );
```

Comment:
No, that's actually a good thing to point out. :guardsman: 

Moai has some very specific guidelines for its formatting. :page_facing_up:  Personally, I think they're kind of silly and ugly. :person_with_blond_hair:  **However**, that on its own is never a good reason to break conventions that were established before you arrived at a codebase.  :computer:  If everybody did that, we'd have no conventions.  :no_good: 

So... I should fix this.  :wrench:  It's tricky, because this is how the current code looks in Xcode:
![how_it_looks_in_xcode](https://cloud.githubusercontent.com/assets/1683575/13445110/2cc580f2-dfbf-11e5-9428-e3b0d0e94f3b.png)

Must be a problem with tab/space conversion.  Let me see if I can figure it out... :radio: